Sources/eppo/ConfigurationStore.swift

```diff
@@ -2,11 +2,20 @@ import Foundation
 
 class ConfigurationStore {
     private var configuration: Configuration?
-    
     private let syncQueue = DispatchQueue(
         label: "com.eppo.configurationStoreQueue", attributes: .concurrent)
     
+    private let fileURL: URL
```

Comment:
minor: Adding persistence to `ConfigurationStore` seems obvious. I'm not sure if that's the best place though. One concern I have is that it adds another responsibility to the store and couples persistence with managing global configuration (so they would be harder to separate in the future). e.g., one place where it shows up is complicating tests with the cleanup step

Questions to think about:
- is configuration caching something that users would want to ever disable?
- is it something that users would want to configure? could there be different sensible strategies?

But I'm probably over-complicating this, so feel free to ignore

Reply:
It's a good thought for sure. There are some downsides, particularly the disk performance latency. Let me consider making this a default and allowing users to disable it.

Reply:
If we factor configuration store into interface, the disk could be an adapter/wrapper (similar to caching assignment logger in python)